index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Sebastián's Portfolio</title>
+</head>
+<body>
+    <div class="wrapper">
+        <nav class="flex-nav">
+            <ul>
+                <li></li>
+                <li><a href="#" class="toggleNav">☰</a></li>
+            </ul>
```

Comment:
I suggest you wrap the hamburger menu in the `nav` tag as stated in the project requirement and remove the `ul` and `li` tags as they are used to designate an unordered list.